README.md

```diff
@@ -1,7 +1,7 @@
 <h2 align="center">🎹 commercetools UI Kit 💅</h2>
 
 <p align="center">
-  <img alt="Logo" src="https://raw.githubusercontent.com/commercetools/ui-kit/master/logo.png" /><br /><br />
+  <a href="https://github.com/commercetools/ui-kit"><img alt="Logo" src="https://raw.githubusercontent.com/commercetools/ui-kit/master/logo.png" /></a><br /><br />
```

Comment:
@YahiaElTai requested there to be a "quick" link to the github repository right on the landing page of https://uikit.commercetools.com/, since he loves consulting it very often 😛 

So I made the big logo become it.

Reply:
Thanks so lovely, Thank you man!